cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/xcm_config.rs

```diff
@@ -123,6 +123,7 @@ pub type XcmOriginToTransactDispatchOrigin = (
 pub struct ParentOrParentsPlurality;
 impl Contains<Location> for ParentOrParentsPlurality {
 	fn contains(location: &Location) -> bool {
+		log::trace!(target: "xcm::contains", "ParentOrParentsPlurality: {:?}", &location);
```

Comment:
```suggestion
		tracing::trace!(target: "xcm::contains", ?location, "ParentOrParentsPlurality contains");
```

should do the same for `xcm_config.rs` of other chains